packages/core/package.json

```diff
@@ -14,19 +14,19 @@
   },
   "license": "MIT",
   "type": "module",
-  "main": "index.js",
-  "module": "./dist/index.es.js",
-  "types": "./dist/index.d.ts",
+  "main": "./index.cjs",
+  "module": "./dist/rx-nostr.js",
+  "types": "./dist/rx-nostr.d.ts",
```

Comment:
Since `rx-nostr.d.ts` is not actually generated, this setting should not work. Here and the following `types` fields should be `./dist/index.d.ts`.

 (But somehow, `tsc` seems to work fine in my local environment. If you know the reason, could you please let us know?)